semgrep-core/src/core/Config_semgrep.atd

```diff
@@ -47,9 +44,18 @@ type t = {
 
   (* assign-patterns (e.g. `$X = $E`) will match var-defs (e.g. `var x = 1;`) *)
   ~vardef_assign <ocaml default="true"> : bool;
+  (* assign-patterns (e.g. `$X = function() { ... }`) will match fld-defs
+   * (e.g. `class Foo {  x = function() { return; } }`).
+   *)
+  ~flddef_assign <ocaml default="false"> : bool;
   (* expression patterns (e.g. `f($X)`) will match attributes (e.g. `@f(a)`)  *)
   ~attr_expr <ocaml default="true"> : bool;
 
+  (* treat arrows (a.k.a short lambdas) like other functions *)
+  ~arrow_is_function <ocaml default="true">: bool;
+  (* treat let/const as equivalent to var (JS-specific) *)
+  ~let_is_var <ocaml default="true">: bool;
```

Comment:
Not mentioned in the changelog, intentionally?

Reply:
It was an old TODO not connected to any issues.
I can add it.